content/blog/2020-03-06-cmake-improvements.md

```diff
@@ -0,0 +1,50 @@
+---
+author: Zack Galbreath
+author-link: https://github.com/zackgalbreath
+date: "2020-03-06T00:00:00Z"
```

Comment:
Set this to the date that the page is actually publish (+/-). Btw, write the date like this:

```yaml
date: 2020-03-16
```

You'll want to update the filename too.

Reply:
Yes, the filename can just be `cmake-improvements.md`. Including the date in the filename is an old Jekyll convention that's no longer necessary.